raglweb/__init__.py

```diff
@@ -284,11 +285,39 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
```

Comment:
PEP8 expects 2 blank lines between function: can you add one above and after this function?

Reply:
I've fixed this and a couple of other PEP8 issues in the code I changed. Are you actually aiming for PEP8 compliance or did you mention this because it breaks the consistency with other methods in this file?

Reply:
Nah just spotted this one. For consistency indeed, don't worry about the rest. Thanks